package.json

```diff
@@ -1,5 +1,28 @@
 {
+  "name": "@keep-network/coverage-pools",
+  "version": "0.0.1",
+  "files": [
+    "contracts/",
+    "!contracts/test/",
+    "deploy/",
+    "deployments/",
+    "export.json"
+  ],
+  "scripts": {
+    "build": "hardhat compile",
+    "deploy": "hardhat deploy --export export.json",
+    "lint": "npm run lint:js && npm run lint:sol",
+    "lint:fix:js": "eslint . --fix",
+    "lint:fix:sol": "solhint 'contracts/**/*.sol' --fix && prettier --write '**/*.sol'",
+    "lint:js": "eslint . ",
+    "lint:sol": "solhint 'contracts/**/*.sol' && prettier --check '**/*.sol'",
+    "test": "hardhat test",
+    "test:system": "NODE_ENV=system-test hardhat test ./test/system/*.test.js",
+    "test:unit:coverage": "hardhat coverage"
+  },
   "dependencies": {
+    "@keep-network/keep-core": "^1.8.0-dev",
+    "@keep-network/tbtc": "^1.1.2-dev",
```

Comment:
In tbtc.js/package.json we use "@keep-network/tbtc": ">1.1.2-pre <1.1.2-rc" range, which excludes packages with `-ropsten` suffix. Here we allow for packages with `-ropsten`. Should we? I don't see clear reason why not, but wanted to double check.

Reply:
Yes, we should be consistent with other repos. We should include just `-dev` packages here. I updated it in 22002144bf07f329609ead9664ec4308f9bb489c.
Please note that for tbtc the range should be `>1.1.2-dev <1.1.2-pre` but the workflow was not yet updated there to publish `dev` packages.